node/resolvers/Queries/Settings.ts

```diff
@@ -1,31 +1,27 @@
 import {
     B2B_SETTINGS_DATA_ENTITY,
-    B2B_SETTINGS_FIELDS,
-    B2B_SETTINGS_SCHEMA_VERSION
+    B2B_SETTINGS_FIELDS
 } from '../../mdSchema'
 import GraphQLError from '../../utils/GraphQLError'
 import checkConfig from '../config'
 
 const B2BSettings = {
-    getB2BSettings: async (_: void, { page,
-        pageSize }: {
-            page: number
-            pageSize: number
-        }, ctx: Context) => {
+    getB2BSettings: async (_: void, {id}: {id: string}, ctx: Context) => {
```

Comment:
I'd use a hardcoded ID like we do in the mutation (it can be exported from there so it's shared) rather than passing it in from the frontend.